### PR TITLE
#6 Update README's to point to wiki setup instructions

### PR DESCRIPTION
**Pull request by kkampardi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,31 @@ Meta repo, used to:
 - have a main repo to attach our waffle board to - https://waffle.io/yunity/yunity
 
 No code here please :)
+## yunity setup
+This provides scripts to get th app up and running. The primary purpose is for
```
Comment (nicksellen) [inline README.md]: little typo here `th` -> `the`

File: README.md [hunk 2]
```diff
@@ -9,6 +9,31 @@ Meta repo, used to:
...
+- understand how to install system dependencies (information only)
+- clone the seperate repos
+- setup the application dependencies
+- setup the database and run the migrations
+- run/manage the application's processes using pm2
+
+The app is split into frontend and backend parts. You can either:
+
+- run everything locally on your machine
+- run the backend in a vagrant vm and run the frontend locally (Note: the app is  in heavy development right now, so this might not work)
```
Comment (nicksellen) [inline README.md]: from `The app is split...` up to this bit could probably be removed now, as the wiki is much more detailed about this

**Comment by nicksellen** [comment]
Thanks! If you just fix those two little bits, I'll merge it in.

**Comment by kkampardi** [comment]
I think its ok now :)

**Pull request merged**